extensions/amp-next-page/1.0/service.js

```diff
@@ -401,45 +421,39 @@ export class NextPageService {
    * @return {!Page}
    */
   createHostPage() {
-    const doc = this.win_.document;
-    const {title, location} = doc;
+    const {title, location} = this.doc_;
     const {href: url} = location;
     const image =
-      parseSchemaImage(doc) || parseOgImage(doc) || parseFavicon(doc) || '';
+      parseSchemaImage(this.doc_) ||
+      parseOgImage(this.doc_) ||
+      parseFavicon(this.doc_) ||
+      '';
     return new HostPage(
       this,
       {
         url,
-        title,
+        title: title || '',
```

Comment:
The type checking still complains here because default values during destructuring only gives default values to undefined variables, nulls are bypassed